hardhat.config.js

```diff
@@ -10,13 +12,35 @@ task("accounts", "Prints the list of accounts", async () => {
   }
 });
 
-// You need to export an object to set up your config
-// Go to https://hardhat.org/config/ to learn more
-
 /**
  * @type import('hardhat/config').HardhatUserConfig
  */
 module.exports = {
-  solidity: "0.8.5",
+  defaultNetwork: "hardhat",
+  networks: {
+    hardhat: {
+      forking: {
+        url: `https://polygon-mainnet.infura.io/v3/${INFURA_API_KEY}`,
+        blockNumber: 15937268,
+      },
+    },
+  },
+  solidity: {
+    version: "0.7.0",
```

Comment:
Let's use latest version of solidity - 0.8.5, Let me know if you think it will create issues. Basically 0.8.x handles math operations overflow itself. So we won't need SafeMath.sol

Reply:
I see you switched to 0.8.0 so I did as well.